features/currency/handling_disabled_currencies.feature

```diff
@@ -0,0 +1,26 @@
+@currencies
+Feature: Handling disabled currencies
+    In order to see right prices
+    As a Customer
+    I want to browse channels only with valid currency
+
+    Background:
+        Given the store operates on a channel named "Web"
+        And that channel operates on "EUR", "USD" and "GBP" currencies
+        And it uses "EUR" currency by default
+        But currency "GBP" is disabled
+
+    Scenario: Not showing the disabled currency
+        When I browse that channel
+        Then it should not have ability to switch current currency to "GBP"
```

Comment:
Then I should not be able to shop in "GBP"
Then I should not be able to pay in "GBP"
Then I should not be able to browse the store with "GBP" as currency

Something along these lines?